src/main/java/teammates/common/datatransfer/FeedbackSessionResponseStatus.java

```diff
@@ -79,28 +79,27 @@ public int compare(String s1, String s2) {
 
     public FeedbackSessionResponseStatus() {
         noResponse = new ArrayList<String>();
-        studentsResponded = new ArrayList<String>();
+        studentsWhoRespond = new ArrayList<String>();
         emailNameTable = new HashMap<String, String>();
         emailSectionTable = new HashMap<String, String>();
         emailTeamNameTable = new HashMap<String, String>();
     }
 
+    // Return students who did not respond to the feedback session.
```

Comment:
`students` -> `list of students` for consistency with the below method. Also, delete the period as the sentence continues on the next line.